nilearn/input_data/base_masker.py

```diff
@@ -270,3 +270,62 @@ def _check_fitted(self):
             raise ValueError('It seems that %s has not been fitted. '
                              'You must call fit() before calling transform().'
                              % self.__class__.__name__)
+
+
+# XXX This function is converging toward filter_and_mask. They will merge
+# eventually.
+def filter_and_extract(imgs, extraction_func,
+                       smoothing_fwhm, t_r,
+                       standardize, detrend, low_pass, high_pass,
+                       confounds, memory, memory_level,
+                       target_fov=None, verbose=0):
+    """Extract representative time series using given function.
+
+    Parameters
+    ----------
+    imgs: images
+        Images to be masked
+
+    extraction_func: function
+        Function used to extract the time series from 4D data.
+
+    kwargs: dict
```

Comment:
There is no kwargs. Maybe you mean to say something like "For all other parameters refer to NiftiMasker documentation"?